iotdb-core/node-commons/src/assembly/resources/conf/iotdb-system.properties

```diff
@@ -1752,6 +1766,13 @@ data_replication_factor=1
 # schema_region_ratis_periodic_snapshot_interval=86400
 # data_region_ratis_periodic_snapshot_interval=86400
 
+####################
+### PipeConsensus Configuration
+####################
+# Default event buffer size for connector and receiver in pipe consensus
+# DataType: int
+# pipe_consensus_pipeline_size=5
```

Comment:
fast_iot_consensus_pipeline_size?

we do not need to expose pipeconsensus to users, we can just put this paramemters behind IoTConsensus

####################
### Fast IoTConsensus Configuration
####################

Reply:
Maybe fix it in further version?